src/commands.ts

```diff
@@ -25,4 +33,66 @@ export function addCommands(plugin: VaultTransferPlugin) {
             insertLinkToOtherVault(editor, view, plugin.settings);
         }
     });
+}
+
+/**
+ * Add a command under the file menu to transfer the current file or folder to another vault.
+ * If a folder is selected, all files in the folder will be transferred.
+ * @param plugin {VaultTransferPlugin} The plugin instance
+ */
+export function addMenuCommands(plugin: VaultTransferPlugin) {
+    plugin.registerEvent(
+      plugin.app.workspace.on("file-menu", (menu, file) => {
+        menu.addItem((item) => {
+          item
+            .setTitle("Vault Transfer")
+            .setIcon("arrow-right-circle")
+            //@ts-ignore
+            const submenu = item.setSubmenu() as Menu;
+            submenu.addItem((subitem) => {
+            subitem
+              .setTitle("Transfer using settings")
+              .setIcon("arrow-right-circle")
+              .onClick(async () => {
+              if (file instanceof TFolder) {
+                transferFolder(file, plugin.app, plugin.settings)
+              } else if (file instanceof TFile) {
+                transferNote(null, file as TFile, plugin.app, plugin.settings);
+              }
+              });
+            submenu.addItem((subitem) => {
+              subitem
+                .setTitle("Transfert in selected folder...")
```

Comment:
Similarly to my other suggestion, I think "Transfer to..." may be enough here
```suggestion
                .setTitle("Transfer to...")
```